### DISCUSSION TITLE
Diagram search

### DISCUSSION THREAD

**Discussion by vedeja** [likec4/likec4#1223] [Ideas]
When browsing my growing collection of diagrams I often have a particular keyword in my head. In Structurizr we can hit the space bar and start typing. Could we have something similar @davydkov?

**Comment by davydkov** [comment]
Hey @vedeja 

That’s a great suggestion. Sometimes I struggle with it too.  
I’ve created an issue #1235 to address it.

Btw, in VSCode you can use `Go to ...` and prefix with `#`/`@`:

![SCR-20241115-sqbo](https://github.com/user-attachments/assets/4028f9f0-cc5e-4a7e-98ff-08da1c36b7cc)

**Reply by vedeja** [reply]
In VSCode it’s a different story, quite easy to find the code. It is while browsing the website I am talking about. Anyway, thanks.

**Comment by vedeja** [comment]
@davydkov I’m trying 1.17.1 now and it works great. Thanks a lot! If anything, an improvement would be to add a keyboard binding to open the search window. Like the space bar, for example.

**Comment by pavelpykhtin** [comment]
Hi, @vedeja 

You can use Ctrl + F shortcut to open the search bar.
Good point that there is no way to find out about them, thanks! I'll add them to the tooltip of the search button.
